src/res/translation/translation_fr_FR.ts

```diff
@@ -1095,7 +1095,7 @@
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="150"/>
         <source>Some sound card drivers do not allow the buffer delay to be changed from within the application. In this case the buffer delay setting is disabled and has to be changed using the sound card driver. On Windows, press the ASIO Device Settings button to open the driver settings panel. On Linux, use the Jack configuration tool to change the buffer size.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Certains pilotes d'interface audio ne permettent pas de modifier la taille de la mémoire tampon depuis l'application. Dans ce cas, ce paramètre est désactivé et doit être modifié à l'aide du pilote de l'interface audio. Sous windows, appuyez sur le bouton Paramètres du périphérique ASIO pour ouvrir le panneau des paramètres du pilote. Sous GNU/Linux, utilisez l'outil de configuration Jack pour modifier la taille de la mémoire tampon.</translation>
```

Comment:
I think you have to remove the type="unfinished" parameter here and on all the other lines.

Reply:
Think it's ok now.